lib/forms/items/formlist.dot

```diff
@@ -1,7 +1,6 @@
 {{# def.partials.formGroup }}
     {{# def.partials.label }}
-    <div ml-bind="MLFormList:{{= it.compName }}"
-              {{? it.disabled }}disabled {{?}}>
+    <div ml-bind="MLFormList:{{= it.compName }}" {{? it.disabled }}disabled {{?}}>
```

Comment:
I was initially talking about the space character after preceding the end `{{?}}` 😄